go/vumitools/billing_worker.py

```diff
@@ -263,7 +275,17 @@ def process_outbound(self, config, msg, connector_name):
                 "Error processing outbound message, sending without billing:"
                 " %r" % (msg,))
             log.err()
-        yield self.publish_outbound(msg, self.receive_inbound_connector, None)
+        if msg is not None:
+            yield self.publish_outbound(
+                msg, self.receive_inbound_connector, None)
+
+    def _handle_credit_cutoff(self, msg):
+        if msg.get('session_event') is not None:
```

Comment:
+1 from @badlard on always modifying the content so please ignore the `close` event part of this suggestion. :)